### PR TITLE
[Review] Apply content type change

### PR DESCRIPTION
**Pull request by pshao25**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/DocumentAnalysisRestClient.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public DocumentAnalysisRestClient(ClientDiagnostics clientDiagnostics, HttpPipel
             _endpoint = endpoint ?? throw new ArgumentNullException(nameof(endpoint));
         }
 
-        internal HttpMessage CreateAnalyzeDocumentRequest(string modelId, ContentType1 contentType, string pages, string locale, StringIndexType? stringIndexType, Stream analyzeRequest)
```
Comment (pshao25) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/DocumentAnalysisRestClient.cs:40]: This file is actually no changes. It is just ordering thing.

File: sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public QueryFormat(QueryFormatType type)
         public JsonTextConfigurationInternal JsonTextConfiguration { get; set; }
         /// <summary> Groups the settings used for formatting the response if the response should be Arrow formatted. </summary>
         public ArrowTextConfigurationInternal ArrowConfiguration { get; set; }
-        /// <summary> Any object. </summary>
+        /// <summary> parquet configuration. </summary>
```
Comment (pshao25) [inline sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs:28]: This file looks a bug fix to me.

File: sdk/synapse/Azure.Analytics.Synapse.AccessControl/src/Generated/RoleAssignmentsClient.cs [hunk 3]
```diff
@@ -106,15 +107,15 @@ public RoleAssignmentsClient(Uri endpoint, TokenCredential credential, AccessCon
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Response> CheckPrincipalAccessAsync(RequestContent content, RequestContext context = null)
```
Comment (pshao25) [inline sdk/synapse/Azure.Analytics.Synapse.AccessControl/src/Generated/RoleAssignmentsClient.cs:109]: This file is DPG. So this breaking change looks by design to me.

File: sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs (hunk 2 above)
Comment (annelo-msft) [reply sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs:28]: I agree.  Could you follow up on this?

Comment (pshao25) [reply sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs:28]: This is from M4 change. Previously it is generated as:
image
now it is:
image

Looking into its [swagger](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e432d9cc87bfed320d8feead4b448be9481c9181/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json). It is:
```
"ParquetConfiguration": {
    "xml": {
      "name": "ParquetTextConfiguration"
    },
    "description": "parquet configuration",
    "type": "object"
  }
```
So this change is by design to me.

File: sdk/synapse/Azure.Analytics.Synapse.Spark/src/Generated/SparkModelFactory.cs [hunk 4]
```diff
@@ -159,7 +159,7 @@ public static SparkSessionCollection SparkSessionCollection(int @from = default,
         /// <param name="result"></param>
         /// <param name="scheduler"></param>
         /// <param name="plugin"></param>
-        /// <param name="errors"></param>
+        /// <param name="errors"> The error information. </param>
```
[the remark this answers is not in the document]
Comment (pshao25) [reply sdk/synapse/Azure.Analytics.Synapse.Spark/src/Generated/SparkModelFactory.cs:162]: It looks a bug.
@timotheeguerin we have below swagger: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/c9992af7235a6550087d4fed8f081ed35019f605/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/sparkJob.json
We can see that `#/definitions/SparkBatchJob/properties/errorInfo/description` is the only place having "The error information." and `#/definitions/SparkSession/properties/errorInfo` don't have any description. 
Why the generated code model has schemas.objects.(!ObjectSchema &ref_177).properties.!Property.(schema: !ArraySchema &ref_147).(elementType: !ObjectSchema &ref_101).properties as below:
image

**Pull request closed without merging**